src/map/script.cpp

```diff
@@ -7942,7 +7940,7 @@ BUILDIN_FUNC(makeitem2) {
 			nameid = UNKNOWN_ITEM_ID;
 	}
 	else
-		nameid = script_getnum(st, 2);
+		nameid = script_getnum64(st, 2);
```

Comment:
possible loss of data from int64 to uint32.

Reply:
🤦 dont know why i changed this